.travis.yml

```diff
@@ -2,10 +2,10 @@ sudo: false
 
 language: php
 php:
-  # PHP 7.4 is not ready yet
-  # - 'master'
-  - '7.3'
-  - '7.2'
+  - nightly
```

Comment:
It is 8.0 actually. I'm not sure that we need it here in the next half year

Reply:
Yeah I agree. I'll remove that one as it's dependent on Zephir anyway.